lib/updaters/types/python.js

```diff
@@ -0,0 +1,29 @@
+const semverRegex = /version[" ]*=[ ]*["'](.*)["']/i
+
+const getVersionIndex = function (lines) {
+  let version
+  const lineNumber = lines.findIndex(line => {
+    const found = line.match(semverRegex)
+    if (found == null) {
+      return false
+    }
+    version = found[1]
```

Comment:
Optional: I had to read this a couple times to see why `version` wasn't always undefined.  I reckon it'd be clearer if it got the index, and then extracted the version in separate steps. This is because people making changes to open source are often in a hurry, so it's worth optimising for ability to skim-read accurately.

Reply:
@TimothyJones , thanks for the feedback.  I updated the variable name from `found` to `versionMatcher` and added a comment as to returning false when no version is found.  Please let me know if you want this adjusted further.